index.html

```diff
@@ -7,7 +7,7 @@
     <link rel="stylesheet" href="styles.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Gelasio:ital,wght@0,400;0,500;1,600&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Fredericka+the+Great&display=swap" rel="stylesheet">
```

Comment:
I personally prefer Gelasio font. From a user experience, I think the font should be simple and very easy to read and so I think Federicka The Great is very pretty but a little too decorative for the task list input. What do you reckon?

Reply:
I think you're right, I was trying to find a typewriter font to match the background, but on closer inspection, it is hard to read sometimes. I've swapped the font back to Gelasio.